test/unit/org/apache/cassandra/utils/binlog/BinLogTest.java

```diff
@@ -182,7 +178,7 @@ public void release()
         });
         t.start();
         t.join(60 * 1000);
-        assertEquals(t.getState(), Thread.State.TERMINATED);
+        assertEquals("BinLog stop should not block forever", t.getState(), Thread.State.TERMINATED);
```

Comment:
nit: I know what you mean here :) but can we make this message clear that BinLog took more than a minute to stop?